src/main/java/com/example/online_shop_api/Service/UserService.java

```diff
@@ -325,7 +325,28 @@ public ResponseEntity<List<UserOrdersResponse>> getCurrentUserOrders(Long userId
         return ResponseEntity.ok(responseList);
     }
 
-    public ResponseEntity<String> changeOrderStatusToCancelled(Long orderId) {
+    private void checkIfOrderHasStatusBasketOrCancelled(Order order) {
```

Comment:
I think that splitting this method into two would be better.

Reply:
Yes, it will improve the readability. I will split it into two methods.